bundle/fix

```diff
@@ -0,0 +1 @@
+../fix/crtm/fix_REL-3.1.1.2/fix
```

Comment:
I know you said this is added to facilitate offline testing of mpasjedi ctests with CRTMv3, but how exactly does this get used? I don't see any build flag to use this instead of the default crtm version.

Reply:
Ah, this is because [CRTMv3](https://github.com/jcsda/CRTMv3) changed the behavior how to decide whether to download CRTM coefficient files from an online URL or use a local copy. If those files exists under bundle/fix, then the CRTMv3 test building step will skip the download.

Reply:
Okay so these are just the coefficient files. So we include these CRTMv3 files in the repo (via link) but still build with v2.4. Does that mean for offline testing, the developer still needs to update `sorc/crtm` on their own?

Reply:
Right, CRTM always has two parts,  source code and coefficient files.

coefficient files are fix files and we added them to facilitate a future upgrade of sorc/crtm to v3 when the satellite experts, such as @xyzemc @HaidaoLin-NOAA think we are ready to do so.

We don't need to do anything at this moment.

Reply:
If we are not updating CRTM source code, then I do not see why we need to add the coefficients? I feel like it would make sense to do it all in one part. Or maybe we should have a separate branch of RDASApp that uses CRTMv3?

Reply:
ok, revert the changes adding the new CRTM fix files.